week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/root.component.ts

```diff
@@ -0,0 +1,28 @@
+import { div } from '../framework/dom-creators'
+import { cartPanel } from './cart-panel.component'
+import {Hero, hero} from './hero.component'
+import {Item} from './cart-item.component';
+
+const items: Item[]  = [
+  {
+    name: 'Tomatoes',
+    amount: 2,
+    unit: 'kg',
+    price: { value: 20, currency: 'PLN' },
+  },
+  {
+    name: 'Banana',
+    amount: 5,
+    unit: 'kg',
+    price: { value: 36, currency: 'PLN' },
+  },
+]
+
+const header: Hero = { title: 'Shopping App', subTitle: 'buy and sell' }
```

Comment:
To trzeba wrzucić od razu jako "props" w linii `:24`, nie ma powodu aby trzymać to oddzielnie?